src/toolbox.ts

```diff
@@ -1970,96 +2160,112 @@ export class KeypointSliderItem extends ToolboxItem {
     public get_html() {
         // Create a SliderHandler instance to handle slider interactions
         const slider_handler = new SliderHandler({
-            "id": this.name.replaceLowerConcat(" ", "-"),
-            "class": "keypoint-slider",
-            "default_value": Math.round(this.filter_value * 100).toString(),
-            "label_units": "%",
-            "slider_event": (slider_value: number) => {
+            id: this.name.replaceLowerConcat(" ", "-"),
+            class: "keypoint-slider",
+            default_value: Math.round(this.filter_value * 100).toString(),
+            label_units: "%",
+            slider_event: (slider_value: number) => {
```

Comment:
This one is particularly interesting as `class` is a reserved word. The LSP in `vscode` doesn't have any issues with it, and the tool still works. :shrug:

Reply:
yeah i suppose for object keys theres no problem shadowing a keyword